lnwallet/interface.go

```diff
@@ -221,6 +223,22 @@ type WalletController interface {
 	// eligible for coin selection.
 	UnlockOutpoint(o wire.OutPoint)
 
+	// LeaseOutput locks an output to the given ID, preventing it from being
```

Comment:
Should really start to split up this interface as we've done for the `keychain` related interfaces lately, but that's outside the scope fo this PR. 